course_discovery/apps/course_metadata/data_loaders/api.py

```diff
@@ -222,6 +222,8 @@ def format_course_run_data(self, body, course=None):
     def format_course_data(self, body):
         defaults = {
             'title': body['name'],
+            'short_description': body['short_description'],
```

Comment:
are we completely sure that the response coming from the server contains always these keys right?